index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio of a King</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+    
+    <!-- container start -->
+    <div class="container relative">
+    
+      <!-- header start -->
+      <div class="wrap header">
+        <header class="flex">
+          <div class="logo">
+            My Logo
+          </div>
+          <div class="ham-icon">
+            <img src="pattern/Union.svg" alt="ham-icon">
+          </div>
+        </header>
+      </div>
+      <!-- header end -->
+
+      <!-- pattern start-->
+      <div class="pattern relative">
+        <div class="x1">
+          <img src="pattern/bigC.svg" alt="pattern">
+        </div>
+        <div class="x2">
+          <img src="pattern/bigR.svg" alt="pattern">
+        </div>
+        <div class="x3">
+          <img src="pattern/smallO.svg" alt="pattern">
+        </div>
+        <div class="x4">
+          <img src="pattern/smallC.svg" alt="pattern">
+        </div>
+        <div class="x5">
+          <img src="pattern/smallR.svg" alt="pattern">
+        </div>
+        <div class="x6">
+          <img src="pattern/small0.svg" alt="pattern">
+        </div>
+        </div>
+        <!-- pattern end -->
+
+        <!-- headline start -->
+       <div class="wrap headline relative">
+         <div class="x7">
+           <img src="pattern/midO.svg" alt="pattern">
+         </div>
+         <h1>I'm Solomon Glad to see you!</h1>
+         <p>
+           I’m a software developer! I can help you build 
+           a product , feature or website Look through some 
+           of my work and experience! If you like what you 
+           see and have a project you need coded, don’t
+           hestiate to contact me.
+         </p>
+     
+         <div class="con">
+           LET'S CONNECT
+        </div>
+
+        <!-- icon start -->
+        <div class="icon flex">
+          <div>
+            <img src="icons/twitter.svg" alt="icon">
+          </div>
+          <div>
+            <img src="icons/space.svg" alt="icon">
+          </div>
+          <div>
+            <img src="icons/git.svg" alt="icon">
+          </div>
+          <div>
+            <img src="icons/chi.svg" alt="icon">
+          </div>
+        </div>
```

Comment:
List the icons with a ul and li tags. Also, wrap the image with an **a** tag. As each icon should navigate to webpage.

## :slightly_smiling_face: